packages/salesforcedx-vscode-core/package.json

```diff
@@ -88,6 +88,13 @@
           "when": "view == sfdx.force.tasks.view"
         }
       ],
+      "editor/context": [
+        {
+          "command": "sfdx.force.source.retrieve.current.file",
```

Comment:
The command appears in the dropdown menu when a user right-clicks anywhere inside of the an open editor.